exporter/otlp-common/Rakefile

```diff
@@ -29,15 +29,21 @@ else
   task default: %i[test rubocop yard]
 end
 
+PROTO_VERSION = 'v0.18.0'
 PROTOBUF_FILES = [
+  'collector/logs/v1/logs_service.proto',
+  'collector/metrics/v1/metrics_service.proto',
+  'collector/trace/v1/trace_service.proto',
   'common/v1/common.proto',
+  'logs/v1/logs.proto',
+  'metrics/v1/metrics.proto',
   'resource/v1/resource.proto',
   'trace/v1/trace.proto',
-  'collector/trace/v1/trace_service.proto'
+  'trace/v1/trace_config.proto'
```

Comment:
Is there some `Dir.glob("./opentelemetry-proto/**/*.proto")` we could rub on this after the proto repository is cloned that would obviate maintaining this list of .proto files by hand in 2 places?

Reply:
Indeed, we could.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/issues/1358